UserInterface.cpp

```diff
@@ -510,16 +509,15 @@ std::optional<LaunchOptions> runControlCenter(const std::wstring& ra3Path, const
 	constexpr auto buttonPadding = 12;
 
 	static constexpr auto clientArea = RECT{0, 0, 800, 600};
-	static constexpr auto allButtons = {

```

Comment:
Did this prevented compilation on MSVC?

Reply:
To be honest I'm quite unfamiliar with C++. According to [](https://stackoverflow.com/questions/16063123/is-it-legal-to-declare-a-constexpr-initializer-list-object) and msvc's error message, I just couldn't find a better way.

Reply:
> To be honest I'm quite unfamiliar with C++. According to and msvc's error message, I guess I just can't find a better way.

@runapp  I just tried to compile, I think the root cause is MSVC think _non-`static` but `constexpr` variables_ still needs to be captures before used (while for gcc it's not needed). That's why MSVC suggested you to use default capture. 
A better way should be revert your changes on capture list (and `constexpr -> const`) first,
then replace those local variables declared as `auto constexpr` to `static auto constexpr`. This will make those variables (or better... constants? 😛 ) usable inside lambda without needing any captures, no matter if it's gcc or MSVC.